test_dependencies/Cargo.lock

```diff
@@ -352,6 +389,20 @@ dependencies = [
  "unicode-ident",
 ]
 
+[[package]]
+name = "tempfile"
+version = "3.3.0"
```

Comment:
Any reason you're not adding the latest version, 3.8.1?

Reply:
Came through as part of the rebase, I've bumped the version now